test/client/spec/components/measure-text.spec.jsx

```diff
@@ -0,0 +1,104 @@
+// Help PhantomJS out
+require("babel-polyfill");
+
+import measureText from "../../../../src/index";
+
+class MockCtx {
+  measureText() {
+    return { width: 42 };
+  }
+}
+
+class MockCanvas {
+  getContext() {
+    return new MockCtx();
+  }
+}
+
+describe("components/measure-text", () => {
+  it("should measure a single line of text", () => {
+    const measurement = measureText({
+      text: "The quick brown fox jumps over the lazy dog",
+      fontFamily: "Helvetica Neue",
+      fontSize: "24px",
+      lineHeight: "1.2",
+      fontWeight: 400,
+      fontStyle: "normal",
+      canvas: new MockCanvas()
+    });
+    expect(measurement).to.have.deep.property("width", 42);
+    expect(measurement).to.have.deep.property("height", `${24 * 1.2}px`);
```

Comment:
Shouldn't the height & width interface be uniform, e.g. always a number or always a string with 'px'? Currently it looks like `width` is a number and `height` is a string with 'px' at the end.